jira_docugen.tex

```diff
@@ -179,28 +286,28 @@ \subsubsection{Test Items}\label{test-items-1}
   DIAObject catalogs (DMS-REQ-0271);
 \end{itemize}
 
-\subsubsection{Requirements}\label{requirements-1}
 
-\begin{itemize}
-\tightlist
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-29}{LVV-29} -
-  DMS-REQ-0069-V-01: Processed Visit Images
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-7}{LVV-7} -
-  DMS-REQ-0010-V-01: Difference Exposures
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-100}{LVV-100} -
-  DMS-REQ-0269-V-01: DIASource Catalog
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-102}{LVV-102} -
-  DMS-REQ-0271-V-01: DIAObject Catalog
-\end{itemize}
 
-\subsubsection{Precondition}\label{precondition-1}
+\subsubsection{Intercase Dependencies}
+LVV-T-17 (AG-00-00)
```

Comment:
But LVV-T-17 has no “intercase dependencies”! This doesn't seem consistent.

Can we adopt the (newer) “predecessors” nomenclature here for clarity?

Reply:
OK, I will update the template in docsteady and the confluence page.